youtube_dl/extractor/bandcamp.py

```diff
@@ -117,7 +120,7 @@ def _real_extract(self, url):
 
         def extract(key):
             return self._search_regex(
-                r'\b%s\s*["\']?\s*:\s*(["\'])(?P<value>(?:(?!\1).)+)\1' % key,
+                r'\b%s\s*["\']?\s*:\s*(["\'])(?P<value>.+)\1' % key,
```

Comment:
No, as this will work incorrectly for empty `""` values.

Reply:
I've made the modifications so it works as expected for empty values and with strings with `"` in the value.